src/formattedcode/format_spdx_tv.py

```diff
@@ -0,0 +1,188 @@
+#
```

Comment:
Two hooks implementations is the way to go IMHO. Otherwise you would introduce coupling between two plugins at the api level.
Note that a format plugin implementation should not know which format was requested (and it should not receive it as an argument) which is something I commented above.

So two hooks implementation functions that call some shared function is fine, having these aware that a certain format has been requested is not.